zio-http/src/test/scala/zhttp/internal/HttpRunnableSpec.scala

```diff
@@ -29,7 +29,7 @@ abstract class HttpRunnableSpec extends DefaultRunnableSpec { self =>
     def run(
       path: Path = !!,
       method: Method = Method.GET,
-      content: String = "",
+      content: HttpData = HttpData.empty,
```

Comment:
Keeping content as `HttpData.empty` looks more logical than defining another overloaded method with default arguments, the compiler would not be able to pick one up for invocations that rely solely on defaults